src/app/core/user-module/user.service.ts

```diff
@@ -56,17 +31,14 @@ export class UserService {
   ): Promise<void> {
     await this.http
       .put(
-        USER_ROUTES.ASSIGN_COLLECTION_MEMBER(collection, memberId),
+        LEGACY_USER_ROUTES.ASSIGN_COLLECTION_MEMBER(collection, memberId),
```

Comment:
This should be access group module and I think we have a route that does this. We shouldn't be using legacy for permissions.